starter-code/components/LowDiv.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+export default function LowDiv() {
+    return (
+        <div id="low-div">
+            <div className="low-div-div">
+                <img className="low-div-img" src="/images/icon1.png" alt="1"/>
+                <h3>Declarative</h3>
+                <p className="text">React makes it painless to create interactive UIs.</p>
+            </div>
```

Comment:
Here you have a repeating structure, you could break it into a component as follows :

```
<InfoBox
        title="Components"
        paragraph="Build encapsulated components that manage their state."
        image="/images/icon2.png"
      />
```